tools/mirror-csv-release/mirror_csv_release.sh

```diff
@@ -287,8 +306,9 @@ mirror() {
 build_and_publish_patched_bundle_image() {
     local bundle_image="${1:?}"
     local dest_prefix="${2:?}"
+    local bundle_dest_prefix="${5:-$dest_prefix}"
     local source_registry=${source_image%\/*}/
-    local dest_image=$(get_dest_image "${bundle_image}" "${dest_prefix}")
+    local dest_image=$(get_dest_image "${bundle_image}" "${bundle_dest_prefix}")
```

Comment:
You patched the wrong function:
on line 218 we are calling `build_and_publish_patched_bundle_image_appregistry` but this is `build_and_publish_patched_bundle_image`.

I'd suggest to:
- keep this
- patch also `build_and_publish_patched_bundle_image_appregistry` with the same logic
- patch the call on line 220
- remove `, used only in appregistry mode` on line 71